.github/scripts/branch/checkBranch.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 branch=${GITHUB_HEAD_REF#refs/*/v}
-if ! [[ $branch =~ [a-z]{3}\/[0-9]+-[A-Za-z-]+$ ]]
+if ! [[ $branch =~ [a-z]{3}\/[0-9]+-[^[:cntrl:]]+$ ]]
```

Comment:
To rethink: Probably we should not allow ALL characters here, for example new line or some strange characters should not be allowed

Reply:
We can think about it in the near future. Thanks to creating a new branch from AS we can't add blank space because AS adds character `_` therefore some cases is not possible 😄